pkg/kubectl/describe.go

```diff
@@ -843,15 +844,6 @@ func describeContainers(label string, containers []api.Container, containerStatu
 			}
 		}
 
-		resourceToQoS := qosutil.GetQoS(&container)
-		if len(resourceToQoS) > 0 {
-			fmt.Fprintf(out, "    QoS Tier:\n")
-		}
-		for _, resource := range SortedQoSResourceNames(resourceToQoS) {
```

Comment:
I can update and remove.  Thanks!

On Sat, Jun 4, 2016 at 7:38 AM, Timo Reimann <notifications@github.com>
wrote:

> In pkg/kubectl/describe.go
> <https://github.com/kubernetes/kubernetes/pull/26568#discussion_r65801510>
> :
>
> > @@ -843,15 +844,6 @@ func describeContainers(label string, containers []api.Container, containerStatu
> >  			}
> >  		}
> >
> > -		resourceToQoS := qosutil.GetQoS(&container)
> > -		if len(resourceToQoS) > 0 {
> > -			fmt.Fprintf(out, "    QoS Tier:\n")
> > -		}
> > -		for _, resource := range SortedQoSResourceNames(resourceToQoS) {
>
> SortedQoSResourceNames()
> <https://github.com/kubernetes/kubernetes/blob/03fc51f74f4228825e0943a3a6e8a097063feb75/pkg/kubectl/sorted_resource_name_list.go#L65>
> isn't used anywhere anymore. Should we also remove it @derekwaynecarr
> <https://github.com/derekwaynecarr>?
>
> Sorry for the late comment -- wasn't aware of this PR until now.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/26568/files/89ff3b25300c39b23d86754cff563ef8b98b6893#r65801510>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AF8dbNx5xsH1ljymPlLEDjkAZqjKC7caks5qIWPDgaJpZM4IqodL>
> .
>